### ISSUE TITLE
Support id and class selectors

### ISSUE THREAD

**Issue by nichtich** [nichtich/Pandoc-Elements#15]
~~~perl
Pandoc::Filter->new( 
    # existing selectors
    Header => sub { ... }, 
    'Suscript|Superscript' => sub { ... },
    # id selector
    'Header#main' => sub { ... },
    # class selector
    'Code.perl' => sub { ... },
    # combination
    'Code.perl.raw' => sub { ... },
    # escaping 
    'Div.some\ class' => sub { ... },
)
~~~

Further extending elements selectors towards something like CSS or even XPointer?

**Comment by nichtich** [comment]
version 0.13 supports undocumented method `Pandoc::Element::match(...)` but the internals should be moved to a class Pandoc::Selector.

**Comment by nichtich** [comment]
Implemented in 0.14 but slow unless code moved to Pandoc::Selector: #18
